types/mastercard-oauth1-signer/mastercard-oauth1-signer-tests.ts

```diff
@@ -0,0 +1,4 @@
+import OAuth = require("./index");
```

Comment:
```suggestion
import OAuth = require("oauth1-signer-nodejs");
```

Reply:
Ah I suspect you mean `mastercard-oauth1-signer` rather than `oauth1-signer-nodejs`